apps/scan/frontend/src/components/misvote_warnings/use_layout_config_hook.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTheme } from 'styled-components';
+
+import { CONFIG } from './constants';
+import { Layout, MisvoteWarningsProps } from './types';
+
+export function useLayoutConfig(props: MisvoteWarningsProps): Layout {
```

Comment:
ah yeah, it's used in both `misvote_warnings` and `warning_details` and it depends on the `useTheme` hook (so, the calculations and result of the hook change when the size theme is changed) - decided to make it a hook as well instead of having clients pass in the theme themselves